javascript/javascript3/week1/giphyApi.js

```diff
@@ -0,0 +1,38 @@
+// 1. I will get button and inputs from html
```

Comment:
Not much to comment on here, good job. I like that you added the comments to break down the problem into smaller solution, that can be very helpful. Variable and function naming is also easy to follow.